DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -5996,13 +5996,15 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CURRENT_PROJECT_VERSION = 1;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
+				DEVELOPMENT_TEAM = HKE973VLUW;
 				INFOPLIST_FILE = DuckDuckGo/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "$(APP_ID)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "Development - App";
```

Comment:
Will these build settings on the app target prevent the `xcconfig` settings from being used?

[DuckDuckGoDeveloper.xcconfig](https://github.com/duckduckgo/iOS/blob/develop/Configuration/DuckDuckGoDeveloper.xcconfig) will try to incorporate settings from external devs so it would be great to make these changes there instead, so that those folks are able to run the app easily. 😄

Reply:
I totally missed the `xcconfig` files and fixed the mismatch from the Xcode's UI level. Thanks for pointing it out.